### PR TITLE
Move ElectrumX and EPS from "Electrum Walllet" to "Docker Plugins"

### LINKED ISSUE #564: Move  Electrum X and Electrum Personal Server to Docker plugins
I propose we move Electrum X and Electrum Personal Server to Docker Plugin section. I think that not many people setting up the wallet for the first time would want to go for this, imo advanced step. It adds quite an overwhelming piece of info for someone who just wants to set up a wallet and start using btcpay server quickly.

This is just a structural/navigational change, nothing more.

docker
docker plugins

Pinging @britttttk and @Kukks for opinion on this one.

If ACK'ed, whoever tackles this issue just needs to move `Electrum X` and `Electrum Personal Server` links in [config.js ](https://github.com/btcpayserver/btcpayserver-doc/blob/master/docs/.vuepress/config.js)to Docker plugins as child's.

### PR DESCRIPTION
**Pull request by Zaxounette**
First PR with me messing with config.js

Moves ElectrumX and EPS from "Electrum Wallet" to "Docker Plugins" as requested in #564

Reviews appreciated.

Closes #564

### CODE REVIEW

**Comment by pavlenex** [comment]
Getting an error when trying to build this locally

```
SyntaxError: Unexpected string
    at wrapSafe (internal/modules/cjs/loader.js:1047:16)
    at Module._compile (internal/modules/cjs/loader.js:1097:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at loadConfig (\node_modules\@vuepress\core\lib\node\loadConfig.js:31:18)
    at App.resolveConfigAndInitialize (\btcpayserver-doc\node_modules\@vuepress\core\lib\node\App.js:64:24)
    at App.process (c\node_modules\@vuepress\core\lib\node\App.js:103:16)
    at dev (\btcpayserver-doc\node_modules\@vuepress\core\lib\index.js:17:13)
    at \node_modules\vuepress\lib\util.js:35:12
    at CAC.<anonymous> (\btcpayserver-doc\node_modules\vuepress\lib\registerCoreCommands.js:38:23)
    at CAC.runMatchedCommand (b\Fork\btcpayserver-doc\node_modules\cac\dist\index.js:718:38)
    at CAC.parse (\node_modules\cac\dist\index.js:638:18)
    at CLI (\node_modules\vuepress\lib\util.js:23:7)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! btcpayserver-docs@1.0.0 start: `vuepress dev docs`
npm ERR! Exit status 1
```

@dennisreimann should be able to direct you what's wrong here.

**Comment by Zaxounette** [comment]
> Getting an error when trying to build this locally
> @dennisreimann should be able to direct you what's wrong here.

I hope so, because of what I understand, there are unexpected strings in files I didn't edit :p

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -154,8 +154,10 @@ module.exports = {
                     path: "/Transmuter/",
                     children: [
                       ["/Transmuter/EmailReceiptsPreset", "Email Receipts Preset"]
-                    ]
+					]  
                   },
+				 ["/ElectrumX", "Electrum X"],
+				 ["/ElectrumPersonalServer", "Electrum Personal Server"]
```
Comment (Zaxounette) [inline docs/.vuepress/config.js:160]:
```suggestion
				 ["/ElectrumPersonalServer", "Electrum Personal Server"],
```

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -154,8 +154,10 @@ module.exports = {
                     path: "/Transmuter/",
                     children: [
                       ["/Transmuter/EmailReceiptsPreset", "Email Receipts Preset"]
-                    ]
+					]
```
Comment (Zaxounette) [inline docs/.vuepress/config.js:157]:
```suggestion
		]  
```

**Pull request merged**